schema.json

```
@@ -35,43 +35,34 @@
     }
   ],
   "created": "2018-04-27",
-  "lastModified": "2023-09-28",
+  "lastModified": "2024-09-16",
   "version": "2.1.1",
   "contributors": [
     {
-      "title": "OpenDataFrance",
+      "title": "Data.Subvention",
       "role": "author"
     },
     {
-      "title": "Pierre Dittgen",
-      "organisation": "Jailbreak",
-      "email": "pierre.dittgen@jailbreak.paris",
-      "role": "author"
-    },
-    {
-      "title": "Quentin Loridant",
-      "organisation": "multi",
-      "email": "quentin.loridant@multi.coop",
-      "role": "contributor"
-    },
-    {
-      "title": "Johan Richer",
-      "organisation": "multi",
-      "email": "johan.richer@multi.coop",
-      "role": "contributor"
-    },
-    {
-      "title": "Amélie Rondot",
-      "organisation": "multi",
-      "email": "amelie.rondot@multi.coop",
+      "title": "Pierlou RAMADE",
+      "organisation": "data.gouv",
       "role": "contributor"
     }
   ],
   "fields": [
+    {
+      "name": "idSubvention",
+      "title": "Identifiant de la subvention",
+      "description": "id interne d'identification d'une subvention permettant de faire le lien avec les actions et permettant de relier les informations sur les différents exercices budgetaire d'une subvention pluriannuel. En cas de financement pluriannuel, cet idéntifiant est le même pour chaque exercice budgetaire.",
```

C'est un id unique par an du coup ?
```suggestion
      "description": "id interne d'identification d'une subvention permettant de faire le lien avec les actions et permettant de relier les informations sur les différents exercices budgétaires d'une subvention pluriannuelle. En cas de financement pluriannuel, cet identifiant est le même pour chaque exercice budgétaire.",
```

---

Oui c'est bien ça :)